components/Branding/LogoSection/LogoSection.js

```diff
@@ -30,12 +30,9 @@ function LogoSection() {
                   <li>
                     <Badge
                       icon={
-                        <Image
-                          src={`${s3}branding/logos/small-blue-logo.png`}
-                          alt="Small Blue Accented Logo"
-                          width={318}
-                          height={60}
-                        />
+                        <div>
+                          <Logo />
+                        </div>
```

Comment:
Why the extra div? It looks like this happens for all uses.

Reply:
An svg is not a block element and therefore only resizes to its parent container size. Putting it in a div allows this to scale with responsive breakpoints.

Reply:
Instead of extra elements, I wonder if we can just make both logos have `display: block`?

Reply:
Quick google says you're right! Then let's wrap the svg with a div in the component itself, so we do it once and don't have to remember the rules around `display` and `svg`.

Reply:
There may be times we want it to not be a block level component and for that it should be decided by the parent how the svg fills who calls it.

Reply:
There's not currently a time. If there is in the future, we can use CSS on the containing div.

Reply:
Components have scoped css, there is no guarantee that having the div in the Logo would make a better experience. I firmly think having the svg wrapped in the div inside Logo would make it more difficult to work with not easier.

Reply:
The CSS isn't scoped. You can still affect CSS across components from one or another component by targeting elements (like `> div`). Even if you don't want to do that, you need only expose a `className` prop to the Logo component attached to the parent div.

Reply:
I added a className to the svg. Can we compromise on having a div in the Logo.js component? You had asked for a way to generate PNG's earlier and this would be one case where knowing that Logo's first element is svg is simpler to handle. It just makes it cleaner to not have html in an svg based component. If we want to `optimize` svg's during build then this is possible also.